newa/cli.py

```diff
@@ -388,12 +285,14 @@ def cmd_schedule(ctx: CLIContext) -> None:
 @main.command(name='execute')
 @click.option(
     '--workers',
-    default=4,
+    default=8,
```

Comment:
I remember our discussion when this default was introduced. Isn't it time to make it something that wouldn't have to be tweaked in the source, e.g. (CPU count / 2), or derive the default from some environment variable?

Reply:
I would really prefer to do this later. IMHO it doesn't make sense to set any limits based on CPU count since those processes are mostly waiting anyway. Moreover, a user can run multiple `newa` instances in parallel anyway.